src/env.h

```diff
@@ -64,6 +56,18 @@ namespace node {
 #define NODE_PUSH_VAL_TO_ARRAY_MAX 8
 #endif
 
+// PER_ISOLATE_* macros: we're going slightly crazy with macros here but the
```

Comment:
I know it’s copy-pasted, but “crazy” isn’t really an appropriate word here… not sure how to phrase it better, but if nothing else comes to your mind, you can go with something like “dark magic”. ;)